<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="LAPTOP-SA9K52P0" name="com.QBBcrm.Maneuver.ValidLoginTest" tests="2" failures="0" timestamp="2021-06-02T11:24:00 IST" time="0.618" errors="1">
  <testcase name="openBrowser" time="0.618" classname="com.QBBcrm.Maneuver.ValidLoginTest">
    <error type="java.io.FileNotFoundException" message="D:\eclipse workspace\Automation\AUTOMATION_FRAMEWORKS\QBBcrm\src\test\resources\Data\Config.properties (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: D:\eclipse workspace\Automation\AUTOMATION_FRAMEWORKS\QBBcrm\src\test\resources\Data\Config.properties (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at com.QBBcrm.genericlib.FileLib.readPropData(FileLib.java:12)
at com.QBBcrm.genericlib.BaseTest.openBrowser(BaseTest.java:14)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- openBrowser -->
  <system-out/>
  <testcase name="loginToApp" time="0.000" classname="com.QBBcrm.Maneuver.ValidLoginTest">
    <skipped/>
  </testcase> <!-- loginToApp -->
  <system-out/>
</testsuite> <!-- com.QBBcrm.Maneuver.ValidLoginTest -->
